src/shared/nested-dropdown-list/nested-dropdown-list-helper.ts

```diff
@@ -43,7 +43,9 @@ export namespace NestedDropdownListHelper {
                     value,
                     expanded: mode === "expand",
                     isSearchTerm: false,
+                    selected: false,
                     checked: false,
+                    indeterminate: undefined,
```

Comment:
why `undefined` instead of `false`?

Reply:
yes. it should `false`, can't recalled why just i put it as `undefined`. 😓